components/KidsList/KidsList.js

```diff
@@ -0,0 +1,32 @@
+import useSWR from "swr";
+import { StyledHeading, StyledList, StyledListContainer,
+    StyledListItem, StyledImage, StyledParagraph } from "./KidsList.styled";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+export default function KidsList() {
+  const { data, isLoading } = useSWR("/api/kids", fetcher);
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  }
+  if (!data) {
+    return <h1>Where are the kids???</h1>;
+  }
+
+  return (
+    <>
+      <StyledHeading>Alle Meilensteine deines Kindes in einer App!</StyledHeading>
+      <StyledParagraph>Für welches Kind möchtest du loggen?</StyledParagraph>
+      <StyledListContainer>
+      <StyledList>
+        {data.map((kid) => (
+          <StyledListItem key={kid._id}>
+          <StyledImage src={kid.image} alt={kid.name} width={500} height={500} /> <br /> {kid.name} <br /> {kid.dob}
```

Comment:
No `<br>` please ^^ make the parent a `flex` column better.

Re `dob`: you can have somewhere in your project a `formatDate` function that takes the timestamp in format `2012-07-22` for instance, and does:

```js
function formatDate(timestamp) {
  return new Date(timestamp).toLocalDateString();
}
```

Then in your components:

```js
<span>{formatDate(kid.dob)}</span>
```

[See docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) for more formatting options.